prow/clonerefs/run.go

```diff
@@ -65,6 +65,10 @@ func (o Options) Run() error {
 		}
 	}
 
+	if p := needsGlobalCookiePath(o.CookiePath, o.GitRefs...); p != "" {
+		configureGlobalCookiefile(p)
```

Comment:
Handle the error.

Reply:
Addressed in https://github.com/kubernetes/test-infra/pull/19231